authors/apps/articles/migrations/0001_initial.py

```diff
@@ -25,6 +25,8 @@ class Migration(migrations.Migration):
                 ('image_url', models.URLField(blank=True, null=True)),
                 ('created_at', models.DateTimeField(auto_now_add=True)),
                 ('updated_at', models.DateTimeField(auto_now=True)),
+                ('createdAt', models.DateTimeField(auto_now_add=True)),
```

Comment:
@gloriaodipo push the latest migrations, without these

Reply:
I deleted the migrations I had and migrated again but there was no change. I'll ask around how to resolve that.

Reply:
@gloriaodipo, you could just delete the migrations file you have and `makemigrations` again. If it leads to a conflict with `develop`, simply keep the latest one you have.